test/loader.test.js

```diff
@@ -20,4 +21,62 @@ describe('Loader', () => {
         expect(src).toMatchSnapshot()
       })
   })
+
+  describe('Watching Deps After An Error', () => {
+    const files = {
+      syntaxError: "css-watching/syntaxError.css",
+      noSyntaxError: "css-watching/noSyntaxError.css",
+      changingFile: "css-watching/styleDep.css"
+    }
+
+    beforeAll(() => copyCssFile(files.noSyntaxError, files.changingFile))
+
+    afterAll(() => deleteCssFile(files.changingFile))
+
+    test('Default', () => {
+      const config = {
+        loader: {
+          options: {
+            plugins: [require("postcss-import")],
+            watching: true
```

Comment:
This should be `config.watch(ing)` instead of `config.loader.watch(ing)` ?

Reply:
Good catch, it isn't needed at all since I'm calling .watch() on the compiler.